README.md

```diff
@@ -240,29 +240,34 @@ The bootloader environment variables *flash-mode* and *flash-mode-devpath* will
 #### Flash Mode 2
 Enable the distribution feature `flash-mode-2` at build time, if you want to use it.
 
-In order to trigger the flash mode 2, use the following commands on the target system:<br>
-```sh
-sudo -s
-bootloader_env.sh set flash-mode 2
-reboot
-...
-Entering omnect flashing mode 2...
-...
-```
-**Note, *bootloader_env.sh* command requires root permissions.**<br>
-
-**Note, `flash-mode 2` is restricted to eth0.**
-
-In the next step, the bmap file and the wic image file have to be transferred, built on the host system:
-```sh
-scp omnect-os-*.wic.bmap omnect@<target-ip>:wic.bmap
-scp omnect-os-*.wic.xz omnect@<target-ip>:wic.xz
-```
-On systems with new openssh clients >= 9.0 you have to use the legacy option when using `scp`. (See [here](https://www.openssh.com/txt/release-9.0) for details.) :
-```sh
-scp -O omnect-os-*.wic.bmap omnect@<target-ip>:wic.bmap
-scp -O omnect-os-*.wic.xz omnect@<target-ip>:wic.xz
-```
+In order to trigger the flash mode 2,
+1. compute the sha256 checksum of your wic.xz
+   ```sh
+   sha256=$(sha256sum image.wic.xz | awk -F" " '{print $1}')
+   ```
+2. use the following commands on the target system:<br>
+    ```sh
+    sudo -s
+    bootloader_env.sh set flash-mode 2
+    bootloader_env.sh set flash-mode-sha256 <computed sha256 from step 1>

```

Comment:
what is the reason to pass sha256 here?
why not pass as file via scp? with this solution we need to know the wic img respectively the chksum before rebooting. (what it makes a bit cumbersome in one of my local scripts where I configure and flash imgs.)

Reply:
mostly identical checksum handling between flash-mode-2 and flash-mode-3.

Reply:
want me to orga a short meeting?

Reply:
i don't want you. it seems we need it,  since it wasn't refined anyway.